spec/unit/jwt/AccessToken.spec.js

```diff
@@ -255,8 +280,9 @@ describe("AccessToken", function () {
     });
 
     it("should create token with multiple grants", function () {
-      var token = new twilio.jwt.AccessToken(accountSid, keySid, "secret");
-      token.identity = "ID@example.com";
+      var token = new twilio.jwt.AccessToken(accountSid, keySid, "secret", {
+        identity: "ID@example.com",
+      });
```

Comment:
Can we just define the token once and use it everywhere, or create it with a function to reduce the duplication?

Reply:
Not possible to define once since we are overriding properties in tests. Created a function to make it better.